src/app/Api.NoSql/Controllers/UsersController.cs

```diff
@@ -48,7 +54,14 @@ public async Task<IActionResult> GetAsync([FromQuery] PageFilterModel model)
         [HttpGet("current")]
         public async Task<IActionResult> GetCurrentAsync()
         {
-            var user = await _userService.FindByIdAsync(CurrentUserId);
+            var user = await _cache.GetAsync<User>(CurrentUserId);
+
+            if (user == null)
+            {
+                user = await _userService.FindByIdAsync(CurrentUserId);
+                await _cache.SetAsync(CurrentUserId, user);
```

Comment:
Let's set not the whole user, but only a few fields, which we don't change (at least for now) - for example, FirstName and LastName. User can update his password, and I'm not sure, if the app will behave correctly with the old password in the cache.